<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgotPass</class>
 <widget class="QMainWindow" name="ForgotPass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1925</width>
    <height>1033</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inventory and Sales Management System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <widget class="QLabel" name="PIC">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1921</width>
      <height>982</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="jj_owner_resources.qrc">:/JJimages/PIC_log_sign_in.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>451</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="jj_owner_resources.qrc">:/JJimages/JJLOGO.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>0</y>
      <width>931</width>
      <height>982</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_xtolanding_forgotpass">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>9</pointsize>
       <weight>87</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 13px; 
padding: 1px; 
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background-color: #374550; 
color: white;</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_forgotPass">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>931</width>
       <height>861</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="enter_contact">
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>901</width>
         <height>521</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>681</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>40</pointsize>
          <weight>87</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #022162;
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background: transparent;</string>
        </property>
        <property name="text">
         <string>Forgot Password</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>150</y>
          <width>761</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>87</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #022162;
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background: transparent;</string>
        </property>
        <property name="text">
         <string>Please enter the contact number associated with your account.</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="enter_contactnum_forgotpass">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>230</y>
          <width>851</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #022162; 
border-radius: 25px; 
padding: 5px;
font-family: &quot;Verdana&quot;, sans-serif; </string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="continueto_entercode">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>360</y>
          <width>851</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>15</pointsize>
          <weight>87</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #374550; 
border-radius: 25px; 
padding: 5px; 
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background-color: #374550; 
color: white;</string>
        </property>
        <property name="text">
         <string>CONTINUE</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="enter_verification">
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>901</width>
         <height>521</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>681</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>40</pointsize>
          <weight>87</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #022162;
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background: transparent;</string>
        </property>
        <property name="text">
         <string>Forgot Password</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>150</y>
          <width>811</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>87</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #022162;
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background: transparent;</string>
        </property>
        <property name="text">
         <string>Enter the code sent to your number.</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="enter_code">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>230</y>
          <width>851</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #022162; 
border-radius: 25px; 
padding: 5px;
font-family: &quot;Verdana&quot;, sans-serif; </string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="confirmCode">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>360</y>
          <width>851</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>15</pointsize>
          <weight>87</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #374550; 
border-radius: 25px; 
padding: 5px; 
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background-color: #374550; 
color: white;</string>
        </property>
        <property name="text">
         <string>CONFIRM</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page">
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>901</width>
         <height>761</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>20</y>
          <width>681</width>
          <height>111</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>40</pointsize>
          <weight>87</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #022162;
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background: transparent;</string>
        </property>
        <property name="text">
         <string>Reset Password</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>761</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>16</pointsize>
          <weight>87</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #022162;
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background: transparent;</string>
        </property>
        <property name="text">
         <string>Set new password</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="setNewPass">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>230</y>
          <width>851</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #022162; 
border-radius: 25px; 
padding: 5px;
font-family: &quot;Verdana&quot;, sans-serif; </string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_confirmPass_reset">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>550</y>
          <width>851</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>15</pointsize>
          <weight>87</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #374550; 
border-radius: 25px; 
padding: 5px; 
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background-color: #374550; 
color: white;</string>
        </property>
        <property name="text">
         <string>CONFIRM</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>340</y>
          <width>761</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>16</pointsize>
          <weight>87</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #022162;
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background: transparent;</string>
        </property>
        <property name="text">
         <string>Confirm new password</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="confirmNewPass">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>410</y>
          <width>851</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #022162; 
border-radius: 25px; 
padding: 5px;
font-family: &quot;Verdana&quot;, sans-serif; </string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="jframe">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>460</y>
      <width>951</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>771</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>24</pointsize>
       <weight>87</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ffffff;
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background: transparent;
</string>
     </property>
     <property name="text">
      <string>J &amp; J Roofsteel and Gutter Supply</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>110</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>18</pointsize>
       <weight>87</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ffffff;
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background: transparent;
</string>
     </property>
     <property name="text">
      <string>Moalboal Branch</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>250</y>
       <width>631</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>16</pointsize>
       <weight>87</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff2bd; 
font-weight: 700;
font-family: &quot;Verdana&quot;, sans-serif; 
background: transparent; 
</string>
     </property>
     <property name="text">
      <string>Supply       |         Install       |         Repair    </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1925</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="jj_owner_resources.qrc"/>
  <include location="jj_owner_resources.qrc"/>
 </resources>
 <connections/>
</ui>
